spring/jta/pom.xml

```diff
@@ -127,6 +122,22 @@
             <version>${version.org.jboss.spec.javax.transaction}</version>
         </dependency>
         <!-- Other dependencies -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-dbcp2</artifactId>
+            <version>2.1.1</version>
```

Comment:
using `${...}`?

Reply:
yeah, I will change to use the version property.